pkg/oonimkall/taskrunner_test.go

```diff
@@ -462,12 +416,9 @@ func TestTaskRunnerRun(t *testing.T) {
 		assertReducedEventsLike(t, expect, reduced)
 	})
 
-	t.Run("with success and InputNone policy", func(t *testing.T) {
+	t.Run("with success and just a single entry", func(t *testing.T) {
 		runner, emitter := newRunnerForTesting()
 		fake := fakeSuccessfulDeps()
-		fake.Builder.MockInputPolicy = func() model.InputPolicy {
-			return model.InputNone
-		}
```

Comment:
We don't need the input policy anymore. We just rely on mocking the target loader in `fakeSuccessfulDeps`.